src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/ChangeSignatureDialog_OutOfProc.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Windows.Automation;
+using Microsoft.VisualStudio.IntegrationTest.Utilities.InProcess;
+using Microsoft.VisualStudio.IntegrationTest.Utilities.Input;
+using Roslyn.Test.Utilities;
+
+namespace Microsoft.VisualStudio.IntegrationTest.Utilities.OutOfProcess
+{
+    public class ChangeSignatureDialog_OutOfProc : OutOfProcComponent
+    {
+        private const string ChangeSignatureDialogAutomationId = "ChangeSignatureDialog";
+
+        public ChangeSignatureDialog_OutOfProc(VisualStudioInstance visualStudioInstance) 
+            : base(visualStudioInstance)
+        {
+        }
+
+        public void VerifyOpen()
+        {
+            var dialog = DialogHelpers.FindDialog(GetMainWindowHWnd(), ChangeSignatureDialogAutomationId, isOpen: true);
+  
+            if (dialog == null)
```

Comment:
Well, I would like FindDialog to have a timeout and crash if it can't find it, but I think I've been outvoted on that. Regardless, this _still_ wouldn't be null. Will remove the check.